### DISCUSSION TITLE
Connection between host and slave

### DISCUSSION THREAD

**Discussion by HelenaRD33** [TheAlbertDev/MASB#58] [Q&A] [answered]
Hi Albert!

From the workflows available in the README, what I understand is that when we start the measurement in the ViSens it has to start the communication with the micro to obtain the data of the CV or CA. However, I don't understand if this communication has to be established directly in the main.c or via the app.  

Thank you in advance.

**Comment by Albert-Alvarez** [comment, marked as answer]
The app only sends measurement requests (CA or CV). It's the microcontroller who sends the measured data to the app automatically (each time a sample is taken) until the measurement is finished.
